generated/api.github.com.json

```diff
@@ -681,19 +692,31 @@
         "PUT /repos/{owner}/{repo}/actions/secrets/{secret_name}"
       ]
     },
-    "secret_scanning_alerts": {
```

Comment:
@timrogers can you verify if this permission was indead removed?

Reply:
Good to know.

I guess we need app-permissions for the different GH* version as well, or add a key stating in which versions the permission is supported? 

For now, I'd suggest to merge and create a follow up issue, so that we are unblocked for further updates

Reply:
I'd agree on that. Makes sense to me.